modelle-718.css

```diff
@@ -142,7 +148,11 @@ table tbody tr td:hover{
     -o-transition: all 300ms;
 }
 
+table tbody tr td img{
+    max-width: 100%;
+}
+
 table tbody tr td p{
```

Comment:
Solche tiefen Verschachtelungen können später zu Spezifitäts-Problemen führen und sind oft unnötig. Was willst du hier treffen?

- alle Absätze in Tabellen? `table p`
- alle Absätze in Tabellen-Inhalten (aber nie im `thead`)? `tbody p`
- alle Absätze in Tabellen-Zellen (aber nie `th`)? `td p`

Davon abgesehen ist die Frage, ob du überhaupt wirklich Absätze in Tabellen-Zellen brauchst, aber das sieht man dem CSS natürlich nicht an.